doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -185,129 +166,118 @@ Newly opened modules and sections inherit the current settings.
+to accessible objects.  (see Section :ref:`invocation-of-tactics`).
 
-   This command displays the type of :n:`@term`. When called in proof mode, the
-   term is checked in the local context of the current subgoal.
+.. cmd:: About @smart_qualid {? @univ_name_list }
 
-   .. cmdv:: @selector: Check @term
+   Displays information about the :n:`@smart_qualid` object, which,
+   if a proof is open,  may be a hypothesis of the selected goal(s),
+   or an accessible theorem, axiom, etc.:
+   its kind (module, constant, assumption, inductive,
+   constructor, abbreviation, …), long name, type, implicit arguments and
+   argument scopes. It does not print the body of definitions or proofs.
 
-      This variant specifies on which subgoal to perform typing
-      (see Section :ref:`invocation-of-tactics`).
+.. cmd:: Check @term
 
+   Displays the type of :n:`@term`. When called in proof mode, the
+   term is checked in the local context of the selected goal(s).
 
 .. cmd:: Eval @red_expr in @term
 
-   This command performs the specified reduction on :n:`@term`, and displays
+   Performs the specified reduction on :n:`@term`, and displays
    the resulting term with its type. The term to be reduced may depend on
-   hypothesis introduced in the first subgoal (if a proof is in
+   hypotheses introduced in the selected goals (if a proof is in
    progress).
 
    .. seealso:: Section :ref:`performingcomputations`.
 
 
 .. cmd:: Compute @term
 
-   This command performs a call-by-value evaluation of term by using the
-   bytecode-based virtual machine. It is a shortcut for ``Eval vm_compute in``
-   :n:`@term`.
+   Evaluates :n:`@term` using the bytecode-based virtual machine.
+   If a proof is open, :n:`@term` may reference hypotheses of the
+   selected goal.  It is a shortcut for :cmd:`Eval` :n:`vm_compute in
+   @term`.
 
    .. seealso:: Section :ref:`performingcomputations`.
 
 
-.. cmd:: Print Assumptions @qualid
-
-   This commands display all the assumptions (axioms, parameters and
-   variables) a theorem or definition depends on. Especially, it informs
-   on the assumptions with respect to which the validity of a theorem
-   relies.
-
-   .. cmdv:: Print Opaque Dependencies @qualid
-      :name: Print Opaque Dependencies
+.. _requests-to-the-environment:
 
-      Displays the set of opaque constants :n:`@qualid` relies on in addition to
-      the assumptions.
+Requests to the environment
+-------------------------------
 
-   .. cmdv:: Print Transparent Dependencies @qualid
-      :name: Print Transparent Dependencies
+.. cmd:: Print Assumptions @smart_qualid
 
-      Displays the set of transparent constants :n:`@qualid` relies on
-      in addition to the assumptions.
+   Displays all the assumptions (axioms, parameters and
+   variables) a theorem or definition depends on.
 
-   .. cmdv:: Print All Dependencies @qualid
-      :name: Print All Dependencies
+   .. exn:: Closed under the global context
```

Comment:
This is the output of the command in case it doesn't depend on any assumption, not an error message (thankfully!).

Reply:
You want a change here?

We have `exn` and `warn` but nothing for simple messages.  Something to add later?

Reply:
Yes, maybe use a note, or some regular text. We would need to see how frequently simple messages need to be documented to decide if that's worth some special directive.

Reply:
Made it a warning, without explanation.  Thought explaining it's only a message would just look dumb (and pointless since IIUC there's no user-relevant difference between a warning and a message except maybe for the color used to print it).  Adding `msg::` is very easy.

Reply:
No, marking this as a warning is not appropriate. Warnings are things people generally want to avoid and there is support to transform warnings into errors. The message could simply be presented in the paragraph. I'm not sure how useful a new directive would be.